modules/deployments-rolling-strategy.adoc

```diff
@@ -60,3 +60,9 @@ These parameters allow the deployment to be tuned for availability and speed. Fo
 
 Generally, if you want fast rollouts, use `maxSurge`. If you have to take into account resource quota and can accept partial unavailability, use
 `maxUnavailable`.
+
+[WARNING]
+====
+The default setting for `maxUnavailable` is `1` for all the machine config pools in {product-title}. It is recommended to not change this value and update one control plane pool at a time. Do not change this value to `3` for the control plane pool.
```

Comment:
There is only one control plane pool. Probably, you mean control plane machinees.

Like this?
`It is recommended to not change this value and update one control plane machine at a time.`

Reply:
good catch, thank you 😄 i will update the admonition